etc/skeleton/driver/normalizer/parser.go

```diff
@@ -5,12 +5,29 @@ import (
 	"github.com/bblfsh/sdk/protocol/native"
 )
 
+// ToNoder specifies the driver options. Driver programmers should fill it
+var ToNoder = &native.ObjectToNoder{}
+
 // ParserBuilder creates a parser that transform source code files into *uast.Node.
-func ParserBuilder(opts driver.ParserOptions) (driver.Parser, error) {
-	toNoder := &native.ObjectToNoder{}
-	parser, err := native.ExecParser(toNoder, opts.NativeBin)
+func ParserBuilder(opts driver.ParserOptions) (parser driver.Parser, err error) {
+	psr, err := native.ExecParser(ToNoder, opts.NativeBin)
 	if err != nil {
-		return nil, err
+		return psr, err
```

Comment:
If using named return values, and `parser` is used instead of `psr`, `return` would be enough here.